src/sfctl/params.py

```diff
@@ -49,12 +49,14 @@ def custom_arguments(self, _): #pylint: disable=too-many-statements
         arg_context.argument('stand_by_replica_keep', type=int)
         arg_context.argument('load_metrics', type=json_encoded)
         arg_context.argument('placement_policy_list', type=json_encoded)
+        arg_context.argument('scaling_policies', type=json_encoded)
```

Comment:
Is it possible to avoid using a json object here? Or does this not scale well at all with the different kinds it has, or potentially any lists it has?

Reply:
Well given that scaling policies is a list of scaling policy which consists of a polymorphic mechanism and a polymorphic trigger - it seems really had to avoid json here :(

If you have any suggestions :)